Inform about how to create the QR code using shell

Kixunil opened the pull request:
People might want to avoid adding another dependencies and compiling stuff for such simple task. The URL can be easily created using shell, so dependencies aren't even needed. This change helps people to use command line, so they don't have to invent it themselves.

ottosuess commented:
is this command missing an `echo '`?

also instead of using `xdg-open` it could be cool to just use `qrencode -t UTF8` to make it work on mac OS as well.

Kixunil commented:
Ah, yeah, it is. Idon't have an experience with `-t UTF8`.

ottosuess commented:
cool 🙌

but i guess most of our users are either on a mac or using ssh to connect to a remote server.
so i don't want to merge `xdg-open` to avoid confusion. rather use `qrencode -t UTF8` to print the qr code to the terminal or don't use `xdg-open` and just generate the image.

michaelWuensch commented:
Please note that your command line does not give the same results as lndconnect.
Yor tls and macaroon are encode in Base64 while lndconnect encodes them in Base64Url

Kixunil commented:
@ottosuess OK, I've removed `xdg-open` completely, because as I found out `-t UTF8` is completely impractical for such huge QR codes. (Changing spec to use fingerprint instead of whole TLS certificate might help.)

@michaelWuensch thanks for finding out! I managed to fix the encoding. I still can't connect, but at least I moved closer. :)

ottosuess commented:
lgtm 🙌

Kixunil commented:
If anyone is wondering, I managed to connect successfully.
